### PR TITLE
Projects frontend

### PR DESCRIPTION
**Pull request by MufaroMakiwa**

### CODE REVIEW

File: src/main/webapp/components/Projects/ConnectionDialog.js [hunk 1]
```diff
@@ -21,6 +26,22 @@ function ConnectionDialog(props) {
...
+            Invite to collaborate?
+          </span>
+
+          <span className="ConnectionDialog_alert"> 
+            {message}
+          </span>
+
+          <div className="Connection_confirm_button_container">
+            <button className="Connection_confirm_button left" onClick={closeDialog}>CANCEL</button>
+            <button className="Connection_confirm_button right" onClick={displayConnectionStatus}>CONTINUE</button>
```
Comment (sarahrittgers26) [inline src/main/webapp/components/Projects/ConnectionDialog.js:41]: Please fix the formatting

```
<button className="Connection_confirm_button right"
        onClick={displayConnectionStatus}>
    CONTINUE
</button>
```

File: src/main/webapp/components/Projects/Projects.js [hunk 2]
```diff
@@ -84,24 +84,57 @@ function Projects() {
...
+
+  // this displays the connection dialog for the user to confirm they want to send the invite
+  const continueProject = (projectId, projectTitle, collaborator, collaboratorId) => {
+    if (isOnline(collaboratorId)) {
+      setOpenConnectionDialog(true);
+      setCurrentConnection({
+        name: collaborator, 
+        userId: collaboratorId
+      });
+      setConnectionAlert(`This will send an invitation to ${collaborator} to continue working on the project, ${projectTitle}`)
```
Comment (sarahrittgers26) [inline src/main/webapp/components/Projects/Projects.js]: please fix the formatting so lines aren't longer than 80 characters, here and below

File: src/main/webapp/components/Projects/Projects.js [hunk 3]
```diff
@@ -84,24 +84,57 @@ function Projects() {
...
+
+    } else {
+      setOpenAlertDialog(true);
+      setAlertWarning(
+        `${collaborator} is currently offline. You can only edit this project when you are both online.`
+        )
```
Comment (sarahrittgers26) [inline src/main/webapp/components/Projects/Projects.js]: nit: please add semicolons

**Review by sarahrittgers26 — APPROVED** [review]
I think the META-INF files should be deleted from the repo

**Review by michaellawes — APPROVED** [review]

**Pull request merged**